ccd-definition/CaseEventToFields/CareSupervision/uploadApplicationDocuments.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadApplicationDocuments",
+    "CaseFieldID": "uploadApplicationDocumentsLabel",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
```

Comment:
could use a better PageID, this will be appended onto the url like this: trigger/uploadApplicationDocuments/uploadApplicationDocuments1

so lets use something else instead of 1